waiter/integration/waiter/basic_test.clj

```diff
@@ -551,6 +551,14 @@
                       (= 200)))
                 :interval 5 :timeout 60)))
 
+        (testing "delete service again (should get 404)"
```

Comment:
Should we wait on all routers to agree that the service has been deleted?

Reply:
Done